ui/pages/confirmations/confirmation/alerts/useTemplateConfirmationAlerts.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Alert,
+  clearAlerts,
+  updateAlerts,
+} from '../../../../ducks/confirm-alerts/confirm-alerts';
+// import { useI18nContext } from '../../../../hooks/useI18nContext';
```

Comment:
delete this and the line below?

Reply:
I am working on a following PR which will add alerts to this and this I left this line.